scala/src/main/org/apache/spark/streaming/api/csharp/CSharpDStream.scala

```diff
@@ -102,7 +102,7 @@ class CSharpDStream(
   override def slideDuration: Duration = parent.slideDuration
 
   override def compute(validTime: Time): Option[RDD[Array[Byte]]] = {
-    val rdd = parent.compute(validTime)
+    val rdd = parent.getOrCompute(validTime)
```

Comment:
Ok, missing test is probably one of the reasons for this bug left uncovered. Can we add a test now?